NuGet.Config

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this change deliberate?

Reply:
Yes, there was some invisible character at the beginning of the file

Reply:
Makes sense. Probably the Unicode BOM.